superset/assets/src/components/TableSelector.jsx

```diff
@@ -127,10 +127,15 @@ export default class TableSelector extends React.PureComponent {
        return SupersetClient.get({ endpoint })
         .then(({ json }) => {
           const filterOptions = createFilterOptions({ options: json.options });
+          const options = json.options.map(o => ({
+            value: o.value,
+            label: o.label,
+            title: o.label,
+          }));
           this.setState(() => ({
             filterOptions,
             tableLoading: false,
-            tableOptions: json.options,
+            tableOptions: options,
```

Comment:
nit: can move the `map` block above here and avoid declaring `const options`

Reply:
thanks @kristw! i updated the code getting rid of the const. I initially did it verbose following the pattern of the previous variable